### DISCUSSION TITLE
How to get past accessibility when building from source?

### DISCUSSION THREAD

**Discussion by TheRoccoB** [linearmouse/linearmouse#325] [Q&A]
Hi @lujjjh, thx for the great app. I wanted to tinker around with it, but when building from source, the app doesn't show up in the Security & Privacy Accessibility window. 

What does your development flow look like? How do I enable accessibility when running from XCode?

**Comment by lujjjh** [comment]
Hey @TheRoccoB,

Please checkout the HEAD of main and this problem should be solved.

The reason you need to regrant accessibility permissions is that your development certificate does not match the release certificate. You need to manually add the application bundle you built to the accessibility permissions. However, the development and release versions share the same bundle identifier, so you must do this again each time you switch between them.

In #326, I chose a separate bundle identifier for development to avoid this problem. Now you can see two LinearMouse items in the list:

<img width="481" alt="image" src="https://user-images.githubusercontent.com/3000535/215262036-b4b65d5e-6e26-406c-8e14-247849e73fb0.png">

**Reply by TheRoccoB** [reply]
Hey, thanks a ton, I'll give it a shot a little later.

**Reply by TheRoccoB** [reply]
Thanks @lujjjh! I think I manually navigated to `~/Library/Developer/Xcode/DerivedData/LinearMouse-aqdgjxbactpkkmfjsgohwepskbme/Build/Products/Debug/LinearMouse.app` and from accessibility and added it (after your recent changes). Seems to work.

In that case I'm able to get accessibility and run the app (tested making the mouse really fast and it worked), but I have questions about the reset code which throws an error:
![image](https://user-images.githubusercontent.com/8771186/215375278-98e0ce04-9c37-45a7-a660-e5041b3db84e.png)

When I type that same command into AppleScript, I get the same error. 
![image](https://user-images.githubusercontent.com/8771186/215375433-a7e6cbd1-6663-497b-8236-f896ae77a581.png)

When I type that command into terminal, it works:

![image](https://user-images.githubusercontent.com/8771186/215375545-0847e410-d7de-481e-aeda-1429a7894018.png)

So my question is, is this meant to be an AppleScript command or a terminal command? Is this just a bug in the code?

**Reply by lujjjh** [reply]
Thanks for reporting it! It's a bug introduced in #202. I'll fix it soon.
